src/Vlumifns_LHApdf.C

```diff
@@ -335,8 +335,9 @@ double qq_12_lumi(const pdfArray& X1, const pdfArray& X2, collider c){
   if (c==pp){ return temp; }
   else if (c==ppbar) { return - temp; }
   else if(c==piso) {
+    // FIXME: here are strange, charm and bottom (anti-)quark PDFs missing
     return - ( 
-	      vuvu * (X1.u - X1.ubar) 
```

Comment:
This line is really worrying, the second parton doesn't have any PDFs...

Reply:
Should be fixed with commit 3ab9b51253d76099a758f8c73439b28367e6acdf. I wonder what impact that bug has. This only matters for proton-isoscalar collisions.